packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -461,6 +463,29 @@ export class CallContext {
     });
   }
 
+  /* @conditional-compile-remove(spotlight) */
+  public setSpotlight(callId: string, spotlightedParticipants: SpotlightedParticipant[]): void {
```

Comment:
I think this is all work in the stateful client PR? should this PR not be pointed at that one?

Reply:
You're right! PR base changed. Thanks!